ci-operator/jobs/openshift/config/openshift-config-master-postsubmits.yaml

```diff
@@ -0,0 +1,36 @@
+postsubmits:
+  openshift/config:
+  - agent: kubernetes
+    context: ""
+    decorate: true
+    name: branch-ci-openshift-config-master-org-sync
```

Comment:
This one is having trouble.  [Job 1][1]: 

```console
$ git fetch https://github.com/openshift/config.git --tags --prune
fatal: could not read Username for 'https://github.com': No such device or address
```

[Job 2][2]:

```console
$ git fetch https://github.com/openshift/config.git --tags --prune
fatal: could not read Username for 'https://github.com': No such device or address
```

[Job 3][3]:

```console
$ git fetch git@github.com:openshift/config.git --tags --prune
No user exists for uid 1000080000
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

[1]: https://storage.googleapis.com/origin-ci-test/logs/branch-ci-openshift-config-master-org-sync/1/build-log.txt
[2]: https://storage.googleapis.com/origin-ci-test/logs/branch-ci-openshift-config-master-org-sync/2/build-log.txt
[3]: https://storage.googleapis.com/origin-ci-test/logs/branch-ci-openshift-config-master-org-sync/3/build-log.txt